ansible_base/resource_registry/resource_server.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime, timedelta
+from typing import TypedDict
+
+import jwt
+from django.conf import settings
+
+from ansible_base.resource_registry.models import service_id
+
+
+class ResourceServerConfig(TypedDict):
+    URL: str
+    SECRET_KEY: str
+    VALIDATE_HTTPS: bool
+    JWT_ALGORITHM: str
+
+
+def get_resource_server_config() -> ResourceServerConfig:
```

Comment:
This function takes no parameters and is just merging a couple defaults with settings. Why not just add the defaults to the settings themselves and requiring it to be there?

Reply:
This is mostly for IDE convenience. Python knows that the function returns a `ResourceServerConfig` dictionary, so most editors can auto complete the key values in the dict.